data/json/items/gun/20x66mm.json

```diff
@@ -74,6 +78,10 @@
     "recoil": 225,
     "durability": 9,
     "burst": 2,
+    "modes": [
+        [ "DEFAULT", "semi-auto", 1 ],
+        [ "BURST", "double", 2 ]
```

Comment:
Whereas using this new syntax to define `[ "AUTO", "2 rd.`, 2 ]` does may be improve the UI somewhat?